content/post/22-03-27-progress-report-32.md

```diff
@@ -0,0 +1,250 @@
++++
+title = "Progress Report #32: Geoff"
+author = "DrSmugleaf, ike709, & Timrod"
```

Comment:
```suggestion
author = "DrSmugleaf, ike709 & Timrod"
```

Reply:
i would understand , and, but , & just looks like ass

Reply:
https://owl.purdue.edu/owl/research_and_citation/apa_style/apa_formatting_and_style_guide/reference_list_basic_rules.html

Reply:
the apa can say whatever they want, its not gonna make it look less like someone collapsed on their keyboard